### ISSUE TITLE
[Feature Request] Control the loss bonus by cvar

### ISSUE THREAD

**Issue by Maxi605** [rehlds/ReGameDLL_CS#842]
Having the ability to modify the loss bonus would be a great addition.

mp_loss_bonus_base - Base of the loss bonus (1.400 by default).
mp_loss_bonus_consecutive_base - Base of the loss bonus if team loss 2 or more rounds consecutively (1.500 by default).
mp_loss_bonus_modifier - Amount of money that gets added to the base per round loss (500 by default).
mp_loss_bonus_maxrounds - Round limit for the bonus modifier to apply (5 by default).
mp_loss_bonus_consecutive_maxrounds - Round limit for the bonus modifier to apply (4 by default).

By default, the values remain the same as the originals, so server admins can choose if they want a different value or not.

**Comment by dystopm** [comment]
@Maxi605 I don't think it is a good idea to have this written in CVars because you'll have to implement all the other RewardRules values also (searching a reasonable way to implement it), which are about 22 account rules, meaning 22 extra CVars 👹  and more useless visual content added to game.cfg file - there must be a criteria when guessing which value export as modifiable

By the way, you can extend this with addons using the `rg_get_account_rules` and `rg_set_account_rules` natives from ReAPI

**Comment by Maxi605** [comment]
> @Maxi605 I don't think it is a good idea to have this written in CVars because you'll have to implement all the other RewardRules values also (searching a reasonable way to implement it), which are about 22 account rules, meaning 22 extra CVars 👹 and more useless visual content added to game.cfg file - there must be a criteria when guessing which value export as modifiable
> 
> By the way, you can extend this with addons using the `rg_get_account_rules` and `rg_set_account_rules` natives from ReAPI

i was going to also add on the suggestion the ability to modify other values but i considered this to be at least the most important features.

I agree to an extent about the "useless visual content", it's really up to each individual server manager to know what they would consider useless. I do agree that it wouldn't be the greatest of ideas to have 22 new cvars added to the file that probably you're going to ignore most of them, that's why i only suggested the ones that are honestly pretty good to have.

**Comment by justgo97** [comment]
Why not just use an amxx plugin for this task?

**Comment by dystopm** [comment]
> Why not just use an amxx plugin for this task?

Not so terrible to implement. An additional config can be added, something like `rewards.cfg` and explain the logic there. It's a hardcoded game logic than can be implemented; thing is, so many cvars. Achievable, need opinions, so I can pull something. @s1lentq @wopox1337 👍👎?

**Comment by Vaqtincha** [comment]
Бред сумасшедшего. Юзай https://dev-cs.ru/resources/470/

**Comment by Nord1cWarr1or** [comment]
Per-reward control already exists for plugins through the ReAPI natives `rg_get_account_rules` / `rg_set_account_rules` (the reward table `m_rgRewardAccountRules` is already in the gamedll). Adding 5+ cvars here would imply the whole ~22 account rules as cvars (as dystopm calculated in the thread), bloat without maintainer approval. Closing as not planned — ReAPI/plugin territory (cf. #233, #599).
